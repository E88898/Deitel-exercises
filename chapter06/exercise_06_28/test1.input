-47.6 54.2 3.1415
