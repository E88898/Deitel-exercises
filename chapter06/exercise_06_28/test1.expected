The smallest value is -47.6
